cypress/e2e/voucher/confirm-your-name.cy.js

```diff
@@ -34,8 +34,7 @@ describe('Confirm your name', () => {
 
         cy.url().should('contain', '/your-name')
         cy.checkA11yApp();
-        cy.get('#f-last-name').clear();
-        cy.get('#f-last-name').type('Smith');
+        cy.get('#f-last-name').invoke('val', 'Smith');
```

Comment:
Useful 👍

Reply:
I think Cypress has got worse in the last version, this test was failing no matter how much I made sure it did this as I wanted. 
<img width="398" alt="image" src="https://github.com/user-attachments/assets/1a5e5fe0-4a6f-4983-aca2-b4bf1a2a1408" />
<img width="424" alt="image" src="https://github.com/user-attachments/assets/f42efc62-0322-4b57-ad9f-ebf8140ab2b1" />
<img width="370" alt="image" src="https://github.com/user-attachments/assets/cda4d84f-f1d4-4d75-a9c0-56396a346674" />
This is just a selection of previously failed runs from today...

Reply:
Downgrade? 😁